### PR TITLE
Fix invalid parameters-template.json and some typos

### PR DESCRIPTION
**Pull request by hosungsmsft**
After this change, I do see that the Moodle resources are created in the MoodleInfrastructure resource group (though the redis cache deployment is failing for its own reason---I observed this frequently).

### CODE REVIEW

File: managedApplication/DeployMoodleManagedApp.md [hunk 1]
```diff
@@ -66,15 +66,20 @@ to manage we'll put these parameter values into environment variables.
...
-adding the `--parameters` attribute. This attribute can take either 
+adding the `--parameters` attribute. This attribute can take either
 a JSON string or a filename (preceded with an '@', e.g. '--parameters @parameters.json`) containing a JSON
 definition for the paramters, e.g.
 
-
-    {
-        "parameterName": "value",
-        "AnOtherParameter"" "another value"
+``` json
```
Comment (rgardler) [inline managedApplication/DeployMoodleManagedApp.md]: These scripts are written to be executed in SimDem. In SimDem code blocks that are denoted by backticks are executable. Therefore this block should be formatted with indents (which is valid md).

File: managedApplication/PublishMoodleManagedApplication.md [hunk 2]
```diff
@@ -88,27 +88,27 @@ The Azure documentation has more information on how to work with [Azure Active D
...
+The following command will add your managed application definition to the Service Catalog.
 
 ``` bash
 az managedapp definition create --name $MOODLE_MANAGED_APP_NAME --location $MOODLE_SERVICE_CATALOG_LOCATION --resource-group $MOODLE_SERVICE_CATALOG_RG_NAME --lock-level $MOODLE_MANAGED_APP_LOCK_LEVEL --display-name $MOODLE_MANAGED_APP_DISPLAY_NAME --description "$MOODLE_MANAGED_APP_DESCRIPTION" --authorizations="$MOODLE_MANAGED_APP_AUTHORIZATIONS" --main-template=@../azuredeploy.json --create-ui-definition=@createUIDefinition.json
 ```
 
 Results:
 
-```
+``` json
```
Comment (rgardler) [inline managedApplication/PublishMoodleManagedApplication.md]: For clarity, given my comment above about executable blocks the exception is blocks that are preceded with "Results:", these are used to pattern match the results of the previous executable block and thus here using backticks is OK.

File: managedApplication/parameters-template.json [hunk 3]
```diff
@@ -1,3 +1,5 @@
 {
-    "sshPublicKey": "GEN-SSH-PUB-KEY"
+    "sshPublicKey": {
```
Comment (rgardler) [inline managedApplication/parameters-template.json]: Well that was one of those "couldn't see it for looking" problems. Good spot.

File: managedApplication/DeployMoodleManagedApp.md (hunk 1 above)
Comment (hosungsmsft) [reply managedApplication/DeployMoodleManagedApp.md]: Got it. Removed.

**Comment by rgardler** [comment]
LGTM

**Comment by hosungsmsft** [comment]
Only fast-forward-merged to the managedapplication branch, anticipating some more changes (addition of params in the UI def. json) before merging to master.

@rgardler - If you think we should merge this to master as is right now, please feel free to go ahead. Thanks!

**Pull request merged**